packages/eslint-plugin-pf-codemods/src/rules/helpers/renameSinglePropOnNode.ts

```diff
@@ -0,0 +1,94 @@
+import { JSXAttribute, JSXIdentifier, JSXOpeningElement } from 'estree-jsx';
+import { RenameConfig } from './renamePropsOnNode';
+import { Rule } from 'eslint';
+
+function isRenameConfig(obj: any): obj is RenameConfig {
+  const hasValidObjectType = typeof obj === 'object' && obj !== null;
+
+  const hasValidNewNameType = typeof obj.newName === 'string';
+
+  const isValidMessage =
+    obj.message === undefined ||
+    typeof obj.message === 'string' ||
+    typeof obj.message === 'function';
+
+  const isValidReplace =
+    obj.replace === undefined || typeof obj.replace === 'boolean';
+
+  return (
+    hasValidObjectType &&
+    hasValidNewNameType &&
+    isValidMessage &&
+    isValidReplace
+  );
+}
+
+const getNewName = (
+  replaced: boolean,
+  propRename: RenameConfig | string | Record<PropertyKey, never>
+) => {
+  if (!replaced) return '';
```

Comment:
This is partially a nit/personal opinion and partially for consistency, but rather than a single line return for the if blocks in these updates maybe we include the curly braces and put the return on a new line.

Not a huge thing, so if nobody else feels strongly about changing this then I'd be fine with it as-is.